app/assets/javascripts/discourse/app/routes/build-group-messages-route.js

```diff
@@ -40,6 +43,12 @@ export default (type) => {
       });
     },
 
+    emptyState() {
+      const title = I18n.t("no_group_messages_title");
```

Comment:
I think this thing could be a one liner rather than storing these two in a short lived variable

Reply:
Sure, I've changed this.